<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sorting</class>
 <widget class="QDialog" name="Sorting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort Options</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>132</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="timeButton">
          <property name="text">
           <string>Chronological (Oldest first)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sizeButton">
          <property name="text">
           <string>File Size (CFDP only) (Smallest first)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="commandButton">
          <property name="text">
           <string>Command (FSW only) (Lowest APID first)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="statusButton">
          <property name="text">
           <string>Status (Successful, Unsuccessful, In Progress)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="timeButton2">
          <property name="text">
           <string>Chronological (Newest first)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sizeButton2">
          <property name="text">
           <string>File Size (CFDP only) (Largest first)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="commandButton2">
          <property name="text">
           <string>Command (FSW only) (Highest APID first)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="statusButton2">
          <property name="text">
           <string>Status (In Progress, Unsuccessful, Successful</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Sorting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Sorting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
